openhtf/io/user_input.py

```diff
@@ -115,7 +118,7 @@ def Respond(self, prompt_id, response):
         self._cond.notifyAll()
 
 
-# Module-level instance to achieve shared prompt state.
+# Module-level instance to achieve shared prompt state. Set by SetupFramework.
```

Comment:
I'm open to doing something better than a module-level instance here if there are better ideas.